src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Changelog
 
-## Unreleased
+## 1.0.0-beta.3
 
+Released 2023-Sep-19
+
+* Added a timeout for `HttpClient` for outgoing call made to metadata service to
+  improve start up time of application.
```

Comment:
improvement is mostly only if running in non-azure vm?

Reply:
clarified